spk/sickchill/src/wizard/install_uifile

```diff
@@ -20,8 +20,5 @@
                 "allowBlank": true
             }
         }]
-    },
```

Comment:
Sorry, this should be kept, to inform the user about the permissing handling.
Only the upgrade wizard is obsolete, as it was ment to inform on upgrades when the original install wizard was without such an information or without the group sc-download (as with DSM5).